application/src/main/java/org/ehrbase/application/abac/CustomMethodSecurityExpressionRoot.java

```diff
@@ -497,14 +545,12 @@ private boolean sendRequestForEach(
      * @param auth Auth object.
      * @return JWT Auth Token
      */
-    private JwtAuthenticationToken getJwtAuthenticationToken(Authentication auth) {
-        JwtAuthenticationToken jwt;
-        if (auth instanceof JwtAuthenticationToken) {
-            jwt = (JwtAuthenticationToken) auth;
+    private AbstractAuthenticationToken getJwtAuthenticationToken(Authentication auth) {
+        if (auth instanceof AbstractAuthenticationToken jwt) {
+            return jwt;
         } else {
             throw new IllegalArgumentException("ABAC: Invalid authentication, no JWT available.");
```

Comment:
Just remove the esle block and throw the exception, but that's really up to you

Reply:
I would rather leave it for now. Code will be changed